storybook/package.json

```diff
@@ -31,11 +35,6 @@
     "*.{jsx,js}": "eslint --cache --fix",
     "*.{js,css,md}": "prettier --write"
   },
-  "husky": {
```

Comment:
please do NOT remove the pre-commit hooks.

Reply:
ok